libraries/botbuilder-applicationinsights/tests/telemetryInitializer.test.js

```diff
@@ -113,4 +120,125 @@ describe(`TelemetryInitializerMiddleware`, function () {
             .assertReply('echo:bar')
             .then(done);
     });
+
+    it('logActivityTelemetry() getter calls logActivityTelemetry function passed into constructor', function() {
+        const logger = new TelemetryLoggerMiddleware(new TelemetryClient('fakeKey'));
+        let passedInFunctionWasCalled = false;
+        const middleware = new TelemetryInitializerMiddleware(logger, () => {
+            passedInFunctionWasCalled = true;
+            return true;
+        });
+
+        const result = middleware.logActivityTelemetry();
+        assert.strictEqual(result, true);
+        assert.strictEqual(passedInFunctionWasCalled, true);
```

Comment:
The expected second value for TelemetryInitializerMiddleware's constructor is a `boolean`, not `() => boolean`. ([link](https://github.com/microsoft/botbuilder-js/blob/ca9de94139f3aa57ff113eb5c87eb62de7873d54/libraries/botbuilder-applicationinsights/src/telemetryInitializerMiddleware.ts#L27))

The [`logActivityTelemetry` getter](https://github.com/microsoft/botbuilder-js/blob/ca9de94139f3aa57ff113eb5c87eb62de7873d54/libraries/botbuilder-applicationinsights/src/telemetryInitializerMiddleware.ts#L35) simply returns the `private readonly _logActivityTelemetry`.

This test has strayed into testing the language 😄 

Was there a reason you decided to test with a function?

```suggestion
        const middleware = new TelemetryInitializerMiddleware(logger, true);

        assert.strictEqual(middleware.logActivityTelemetry, true);
```

Reply:
This is one of those tests that I only really added for the 100% code coverage. Since `_logActivityTelemetry` can only be set by passing it into the constructor, there's no way of asserting that it is a particular value before it's instantiated. Meaning, I can't:

```js
assert.strictEqual(middleware.logActivityTelemetry, undefined);

Reply:
That code coverage mechanism is unpleasant, thanks for providing context!